pkg/config/draftconfig.go

```diff
@@ -68,28 +69,60 @@ func (d *DraftConfig) GetNameOverride(path string) string {
 	return prefix
 }
 
-// ApplyDefaultVariables will apply the defaults to variables that are not already set
-func (d *DraftConfig) ApplyDefaultVariables(customInputs map[string]string) error {
-	varIdxMap := VariableIdxMap(d.Variables)
+func (d *DraftConfig) AddVariable(name, value string) {
```

Comment:
how do we guard against duplicates? we should either return an error, or change to something idempotent like PutVariable() that would update existing/create

Reply:
otherwise this will make multiple buildervars with the same name, while getVariable will only return the last one